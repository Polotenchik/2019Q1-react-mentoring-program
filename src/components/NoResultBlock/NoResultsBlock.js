import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.div`
    min-height: 110px;
    width: inherit;
    text-align: center;
    font-size: 40px;
    padding-top: 200px;    
`;


const NoResultsBlock = () => (
    <Wrapper>
        No films found
    </Wrapper>
);

export default NoResultsBlock;
