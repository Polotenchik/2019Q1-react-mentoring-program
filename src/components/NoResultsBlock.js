import React from 'react';

export const NoResultsBlock = () => {
    return (
        <div className={'no-results'}>
            No films found
        </div>
    );
};