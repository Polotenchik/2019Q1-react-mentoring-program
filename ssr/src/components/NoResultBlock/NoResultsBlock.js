import React from 'react';

export const NoResultsBlock = () => (
        <div className={'no-results'}>
            No films found
        </div>
);
