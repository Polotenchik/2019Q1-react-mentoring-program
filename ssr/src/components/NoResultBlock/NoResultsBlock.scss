.no-results {
    min-height: 110px;
    width: inherit;
    text-align: center;
    font-size: 40px;
    padding-top: 200px;
}

.no-found {
    min-height: 110px;
    width: inherit;
    color: white;
    text-align: center;
    font-size: 40px;
    padding-top: 200px;
}