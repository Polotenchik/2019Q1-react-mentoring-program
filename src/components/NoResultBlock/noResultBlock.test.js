import React from 'react';
import { shallow } from 'enzyme';
import { NoResultsBlock } from './NoResultsBlock';

describe('<NoResultsBlock />', () => {
  describe('<NoResultsBlock />', () => {
    it('should render NoResultBlock', () => {
      const wrapper = shallow(<NoResultsBlock />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
