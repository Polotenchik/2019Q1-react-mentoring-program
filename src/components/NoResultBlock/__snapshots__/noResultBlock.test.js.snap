// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoResultsBlock /> <NoResultsBlock /> should render NoResultBlock 1`] = `
<div
  className="no-results"
>
  No films found
</div>
`;
